app/assets/stylesheets/custom.scss

```diff
@@ -342,7 +357,7 @@ font-size: 40px;
   height: 240px;
 }
 
-#map {
+#map_1 {
```

Comment:
Avoid using id selectors<br>Selector `map_1` should be written in lowercase with hyphens